src/gui/gtk.c

```diff
@@ -2627,8 +2628,18 @@ void dt_gui_load_theme(const char *theme)
   //set font size
   if(dt_conf_get_bool("use_system_font"))
     gtk_settings_reset_property(gtk_settings_get_default(), "gtk-font-name");
-  else {
-    gchar *font_name = dt_util_dstrcat(NULL, _("Sans %.1f"), dt_conf_get_float("font_size"));
+  else 
+  {
+    //font name needs to include a comma or period as the decimal separator
+    //depending on locale setting
+    const gchar *font_size = dt_util_dstrcat(NULL, _("%.1f"), dt_conf_get_float("font_size"));
```

Comment:
Sorry I was't clear, this line create a float number with a comma or dot depending on the locale. In France this is generating a comma. And CSS only supports a . so the proper fix is to always do:

`      font_name = dt_util_dstrcat(NULL, _("Sans %s"), dt_util_str_replace(font_size, ",", "."));`

That is changing comma to dot. You can do that conditionally if you prefer but you must ensure that the font_name is always using a dot.

Reply:
Thanks @TurboGit I was just coming to that realisation (hard without testing myself). I'll try again!